operator/src/main/resources/scripts/utils.sh

```diff
@@ -791,9 +791,9 @@ checkCompatibilityModeInitContainersWithLegacyAuxImages() {
     rm -f ${AUXILIARY_IMAGE_PATH}/testaccess.tmp || return 1
 
     # The container .out files embed their container name, the names will sort in the same order in which the containers ran
-    out_files=$(ls -1 $AUXILIARY_IMAGE_PATH/compatibilityModeInitContainerLogs/*.out 2>/dev/null | sort --version-sort)
+    out_files=$(ls -1 "${AUXILIARY_IMAGE_PATH}"/"${AUXILIARY_IMAGE_COMMAND_LOGS_DIR}"/*.out 2>/dev/null | sort --version-sort)
```

Comment:
Can it be simplified "${AUXILIARY_IMAGE_PATH}/${AUXILIARY_IMAGE_COMMAND_LOGS_DIR}/*.out" ? instead of multiple double quotes in between?

Reply:
Updated. Thanks!